### DISCUSSION TITLE
535 Invalid credential

### DISCUSSION THREAD

**Discussion by manmln** [postalserver/postal#3239] [Help with using Postal]
Hello,
I installed Postal using instructions and try to test sending emails.
Opened telnet
type EHLO command - get 250-My capabilities are
250 AUTH CRAM-MD5 PLAIN LOGIN
type AUTH LOGIN command- get 334 VXNlcm5hbWU6
type username copied from Sending E-mails page - get 334 UGFzc3dvcmQ6
type password copied from Sending E-mails page - get 535 Invalid credential

No matter what else I do (created new key, tried base64 encryption) I get invalid credential.
Please advise,
Michael
![postal](https://github.com/user-attachments/assets/6461a8a8-1ed5-4466-b96d-50ff9a4095d6)

**Comment by willpower232** [comment]
Both auth plain and auth login need base64 so its hard to test manually as its very easy to encode some extra whitespace at the end of the value being encoded.

You may get on better with a real mail software library, or even a desktop client will be able to send the message, just not receive one.

**Comment by manmln** [comment]
I tried entering both encrypted and unencrypted username/password in Mautic - get connection timeout.
I also tried to encrypt using 2 different tools and get the same string.
My username was generated as mr/myregistry-info. Are there any illegal characters?

**Reply by willpower232** [reply]
Presumably you would be entering unencrypted information into Mautic and it would do that part itself so there must be something separate which prevents it connecting to your Postal.

Are you able to see any logs or any raw output for what Mautic is trying to do?

**Comment by manmln** [comment]
I rebuilt the server and reinstalled Postal. Same result. I also tried to use another client, and it doesn't work either. Has it anything to do with my configuration file postal.yml? I noticed that there are 2 fields: username and password that are supposed to be populated later. Can it be the issue?
Thanks

**Reply by willpower232** [reply]
> get connection timeout

I think you should deal with this first. Are you able to see any information from Mautic?
